README.md

```diff
@@ -46,7 +51,7 @@ range of applications using the [Google Maps SDK for iOS][sdk].
   ```
 
 > [!IMPORTANT]
-> You also need to install the Maps SDK for iOS, which is also supported in Swift Package Manager at the URL `https://github.com/googlemaps/ios-maps-sdk`
+> You also need to install the Maps SDK for iOS, which is also supported in Swift Package Manager at the URL `https://github.com/googlemaps/google-maps-ios-utils`
```

Comment:
Here is talking about the Google Maps iOS SDK, so the url was correct, shouldn't be changed.

Reply:
Good catch... FIXED